### ISSUE TITLE
Can't log into Android app

### ISSUE THREAD

**Issue by rbmattis** [keybase/client#7077]
I was logged into the android app, when I tried to open it, it made me relog in, this was the error message:
![dsc01947](https://cloud.githubusercontent.com/assets/28842202/26284803/7bbc8ad2-3e11-11e7-9968-d2bab49058e2.JPG)

I uninstalled-reinstalled the app, same error.

I revoked my device "phone", uninstalled-reinstalled, same error. 

I'm 100% confident that the passphrase is correct.

**Comment by maxtaco** [comment]
Sorry about this, it's definitely our bug.

@oconnor663 it's due to your new sigchain-reset sensing code, at least the error is raised there...

**Comment by rbmattis** [comment]
is there anything to do on my end besides wait?

**Comment by maxtaco** [comment]
I don't believe so, but it's a high priority bug so hopefully we can fix it ASAP

thanks for your patience and help

**Comment by maxtaco** [comment]
Closed in #7079.  Should be fixed in the next release.

**Comment by mannp** [comment]
Having the same issue, but it seems fixed now, which is great :) 

Do you have a beta provisioning through f-droid or google play, so that we can pick up the build before a formal release?

Cheers and love the apps!

**Comment by rbmattis** [comment]
what @mannp said. Do you have a nightly apk I could try?
